tests/GraphNodes/GraphObjectFactoryTest.php

```diff
@@ -235,203 +73,42 @@ public function testAGraphObjectWillBeCastAsAGraphObject()
         ], $graphData);
     }
 
-    public function testAGraphObjectWithARootDataKeyWillBeCastAsAGraphObject()
+    public function testAListFromGraphWillBeCastAsAGraphEdge()
     {
         $data = json_encode([
-            'data' => [
-                'id' => '123',
-                'name' => 'Foo McBar',
-                'link' => 'http://facebook/foo',
-            ],
+          'data' => [
+            [
+              'id' => '123',
+              'name' => 'Foo McBar',
+              'link' => 'http://facebook/foo',
+            ],
+            [
+              'id' => '1337',
+              'name' => 'Bar McBaz',
+              'link' => 'http://facebook/bar',
+            ],
+          ],
+          'paging' => [
+            'next' => 'http://facebook/next',
+            'previous' => 'http://facebook/prev',
+          ],
         ]);
-
         $res = new FacebookResponse($this->request, $data);
 
         $factory = new GraphObjectFactory($res);
-        $graphObject = $factory->makeGraphObject();
-        $graphData = $graphObject->asArray();
+        $graphEdge = $factory->makeGraphList();
```

Comment:
This is to show BC but renamed `$graphEdge` to `$graphList` to be more explicit what it's testing. :)